package.json

```diff
@@ -24,6 +24,7 @@
         "@vscode/webview-ui-toolkit": "^1.4.0",
         "antd": "^5.22.1",
         "primeflex": "^3.3.1",
```

Comment:
The `primeflex` story isn't entirely clear to me. I see it as a dependency here but no other reference in this repo (except from the lock file).

Reply:
I just looked into it, and we are not using it anymore.